code/game/objects/items.dm

```diff
@@ -67,8 +67,16 @@
 	//Does not affect damage dealt to mobs
 
 /obj/item/Destroy()
-	qdel(hidden_uplink)
-	hidden_uplink = null
+	QDEL_NULL(hidden_uplink)
+	QDEL_NULL(blood_overlay)
+	origin_tech = null
```

Comment:
afaik many of these are just strings or lists of strings, there's no need to do this.

examine each one carefully, strings and numbers can be left alone, only references to datums need to be nulled